charts/aws-cloud-controller-manager/values.yaml

```diff
@@ -0,0 +1,116 @@
+namespace: "kube-system"
+
+extraArgs: {}
+
+
+image:
+    repository: gcr.io/k8s-staging-provider-aws/cloud-controller-manager
+    tag:  v1.19.0-alpha.1
+
+# nameOverride -- String to partially override `aws-cloud-controller-manager.fullname`
+nameOverride: "aws-cloud-controller-manager"
+
+# nodeSelector -- Node labels for pod assignment. Ref: https://kubernetes.io/docs/user-guide/node-selection/.
+
+nodeSelector:
+  node-role.kubernetes.io/master: ""
+
+clusterRoleRules:
```

Comment:
what is benefit of putting in values versus embedding in yaml?

(I don't have much opinion on putting it in one place or the other, just wondering.)

Reply:
I thought to add values that are KV as embedded for easy readability, Do you want to change. I think this looks good for a few values such as for tolerations, resources?

Reply:
I'm okay with it, no need to change.

Reply:
Have a new squashed commit that resolves all comments you have added, kindly please review and let me know